### PR TITLE
Feature/budget dom updates

### PR DESCRIPTION
**Pull request by fernandorobles97**
What I did:
- Added budget input form elements in HTML. 
- Added query selectors for form elements, and event listener.
- Added function to render recipes based on if it meets the budget the user submits.
- Added pointer cursors for checkboxes and submit button.

Did this break anything?
- [x] No
- [ ]  Yes

Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Add new GIF to readme

### CODE REVIEW

**Review by hvohr — APPROVED** [review]
this looks incredible !! nice clean code and amazing use of the 'required' for the input

File: dist/index.html [hunk 1]
```diff
@@ -48,7 +48,8 @@ <h1 class="title" tabindex="0">What's Cookin?</h1>
   <div class="all-recipes-wrapper">
     <aside class="all-filters hidden">
       <div class="filter-option">
-        <form class="input-budget">
+        <form class="input-budget" required>
+          <input type="submit" class="submit" value="Submit">
```
Comment (Denzel-Turambi) [inline dist/index.html:52]: Nice inclusion of a submit input

File: src/domUpdates.js [hunk 2]
```diff
@@ -145,6 +146,16 @@ savedRecipeDisplay.addEventListener('click', event => {
   }
 });
 
+budgetInputForm.addEventListener('submit', event => {
```
Comment (andreasorensen) [inline src/domUpdates.js:149]: I like the isNan usage!

**Review by andreasorensen — APPROVED** [review]

File: src/domUpdates.js [hunk 3]
```diff
@@ -253,6 +263,24 @@ const showSavedRecipes = (array) => {
...
+    <p class="recipe">${recipe.name}</p>
+    </div>`)
+  if (!recipeFromCost.length) {
+    allRecipeDisplay.innerHTML = `
+    <div class="no-recipe-found-message">
+      <p>Sorry, ${currentUser.name}, no recipes meet this budget.</p>
+    </div>`
+  }
+}
+
```
Comment (Denzel-Turambi) [inline src/domUpdates.js:283]: Love this function

**Review by Denzel-Turambi — APPROVED** [review]

**Pull request merged**